root/artist/header.tt

```diff
@@ -1,4 +1,4 @@
-<div class="artistheader">
+<div class="artistheader [% artist.type_name.lower() _ '-icon' IF artist.type.name %]">
```

Comment:
In the same idea, note that `.name` at the end of `IF artist.type.name` is unnecessary and thus should preferably be removed. Indeed, `artist.type.name` is undefined if and only if `artist.type` is undefined. See [`type` and `type_name` definitions](https://github.com/dpmittal/musicbrainz-server/blob/master/lib/MusicBrainz/Server/Entity/Role/Type.pm#L22-L30) for reference.